test/observer/toast.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:flutter/services.dart';
+
+import '../test.dart';
+
+/// An observer for toast messages from the flutter toast package.
+class ToastChannelObserver {
+  /// The method channel used by the flutter toast package
+  static const MethodChannel _channel = MethodChannel('PonnamKarthik/fluttertoast');
+  Map<String, dynamic>? _lastToastArguments;  /// The arguments for the last requested toast message.
+  String? get lastToastMessage {
+    final lastArguments = _lastToastArguments;
+    clearLastToast();
```

Comment:
Why do we need to clear it here (same in the other observer)?

Reply:
See my comment above for reasoning.